### PR TITLE
Temperature support

### PR DESCRIPTION
**Pull request by IgorAnohin**
Add page with tutorials:
- Temperature monitoring
- Changing target temperature

### CODE REVIEW

File: docs/qgc/sensor-temperature.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+!!! attention
+    Please don’t set temperature below and much more than environment as we can't heat the sensors endlessly, likewise their temperature can't be lower than environment's.
+
+<div style="text-align: center;"><img src="../../img/qgc/temperature/new-target-temp-parameter.png"></div><br>
+
+After that your board temperature will gradually change. It takes about 1-2 minutes.
+
+!!! attention
+    Calibrate your sensors only after temperature stabilize.
```
Comment (staroselskii) [inline docs/qgc/sensor-temperature.md]: stabilizes

**Comment by AlexeyBulatov** [comment]
Please add some description for what we need IMU heater.

**Comment by AlexeyBulatov** [comment]
Also add something about min/max tempreture

**Comment by AlexeyBulatov** [comment]
> is bad for sensors data

Temperature influences on bias change.

> Also add something about min/max temperature

I mean max/min value of BRD_IMU_TARGTEMP. Max is defined, min > temp of the environment.

File: docs/qgc/sensor-temperature.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+##Intro
+
+Changing temperature of sensors during the flight influences on bias change. So we use the heating to maintain a stable temperature. It allows to get more accurate data. Using QGC you can not only monitor temperature but also change it.
```
Comment (staroselskii) [inline docs/qgc/sensor-temperature.md]: I'm confused by the phrasing.

Maybe something like this?
"Temperature fluctuations can influence the data that is acquired from sensors on the board"

File: docs/qgc/sensor-temperature.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+<div style="text-align: center;"><img src="../../img/qgc/temperature/parameters.png"></div><br>
+
+Fill the `Search` with "BRD_IMU_TARGTEMP" string and click on the found parameter. You will see `Parameter Editor`.
+
+<div style="text-align: center;"><img src="../../img/qgc/temperature/target-temp-parameter.png"></div><br>
+
+Enter the desired temperature and press `Save`:
+
+!!! attention
+    Please don’t set temperature below and much more than environment as we can't heat the sensors endlessly, likewise their temperature can't be lower than environment's. Therefore **min temperature > temperature of environment, max temperature < +80 ºC**.
```
Comment (staroselskii) [inline docs/qgc/sensor-temperature.md]: don't set the temperature to something that is very different from the environment

**Pull request merged**